internal/backend/gs/config.go

```diff
@@ -16,13 +16,15 @@ type Config struct {
 	Bucket    string
 	Prefix    string
 
-	Connections uint `option:"connections" help:"set a limit for the number of concurrent connections (default: 5)"`
+	Connections uint   `option:"connections" help:"set a limit for the number of concurrent connections (default: 5)"`
+	Region      string `option:"region" help:"region for the bucket (default: us)"`
```

Comment:
From what I understand, the region is only used when creating a bucket. That should be reflected in the help text.